src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
 import ScreenWrapper from '@components/ScreenWrapper';
+import Search from '@components/Search';
+import useActiveRoute from '@hooks/useActiveRoute';
 import useLocalize from '@hooks/useLocalize';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import TopBar from '@navigation/AppNavigator/createCustomBottomTabNavigator/TopBar';
 import SearchFilters from './SearchFilters';
 
-// import EmptySearchView from './EmptySearchView';
-
 function SearchPageBottomTab() {
     const {translate} = useLocalize();
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const activeRoute = useActiveRoute();
+
+    const currentQuery = activeRoute?.params && 'query' in activeRoute.params ? activeRoute?.params?.query : '';
+    const query = String(currentQuery);
 
     return (
-        <ScreenWrapper testID={SearchPageBottomTab.displayName}>
+        <ScreenWrapper
+            testID={SearchPageBottomTab.displayName}
+            style={[{paddingTop: 0}]}
```

Comment:
Replace the inline style with styles.pt0

Reply:
will remove for now because this specific padding fix will be added in a separate PR done by Carlos